### DISCUSSION TITLE
better showOffsets

### DISCUSSION THREAD

**Discussion by glaprida** [GrapesJS/grapesjs#4999] [Ideas]
Aside from the fact that currently `showOffsetsSelected` is not working ( https://github.com/GrapesJS/grapesjs/issues/4998 ) I have some ideas for a better `showOffsets` functionality.

It would be great if it could also show the `gap` property between grid items and flex children as you hover or select the `display:grid` or `display:flex` parent.

Also, it would be great if `margin-inline:auto` (margin-left and margin-right auto) could be displayed as a margin from the element bounds all the way to the parent container.

**Comment by artf** [comment]
Yeah that would be nice, I think current offset helpers are probably too much hard coded and could be definitely improved to be more customizable. I'd be glad to accept PRs in regards.
